### PR TITLE
Update investment requirements to add another

### PR DESCRIPTION
**Pull request by ztolley**
Adds 'add another' support to the investment requirements form forstrategic drivers, competitor countries and possible uk regions.

screencapture-localhost-3000-investment-projects-d886a660-5268-4149-823b-5805029e0639-edit-requirements-1509030249644

screencapture-localhost-3000-investment-projects-d886a660-5268-4149-823b-5805029e0639-details-1509030274533

### CODE REVIEW

File: src/apps/investment-projects/labels.js [hunk 1]
```diff
@@ -45,16 +45,16 @@ const labels = {
...
       competitor_countries: 'Competitor countries',
       uk_region_locations: 'Possible UK locations',
       uk_company: 'UK recipient company',
     },
     edit: {
-      strategic_drivers: 'Main strategic driver behind this investment',
+      strategic_drivers: 'Strategic drivers behind this investment',
       client_considering_other_countries: 'Is the client considering other countries?',
-      uk_region_locations: 'Possible UK location for this investment',
+      uk_region_locations: 'Possible UK locations for this investment',
```
Comment (feedmypixel) [inline src/apps/investment-projects/labels.js]: should this be `Possible UK location(s)` or `Possible UK location/s`

File: src/apps/investment-projects/labels.js [hunk 2]
```diff
@@ -45,16 +45,16 @@ const labels = {
...
-      strategic_drivers: 'Main strategic driver',
+      strategic_drivers: 'Strategic drivers',
       client_requirements: 'Client requirements',
       competitor_countries: 'Competitor countries',
       uk_region_locations: 'Possible UK locations',
       uk_company: 'UK recipient company',
     },
     edit: {
-      strategic_drivers: 'Main strategic driver behind this investment',
+      strategic_drivers: 'Strategic drivers behind this investment',
```
Comment (feedmypixel) [inline src/apps/investment-projects/labels.js]: should this be `Strategic driver(s)` or `Strategic driver/s`

File: src/apps/investment-projects/middleware/forms/requirements.js [hunk 3]
```diff
@@ -18,20 +17,32 @@ function populateForm (req, res, next) {
   next()
 }
 
+// Strips out empty entries so they are not posted (which result in errors)
+// and when an error is thrown it doesn't mean yet another is added
+function cleanArray (values) {
+  return flatten([values])
+    .filter(item => item)
+}
```
Comment (feedmypixel) [inline src/apps/investment-projects/middleware/forms/requirements.js]: feels like `values` could become `array`?

**Comment by feedmypixel** [comment]
@ztolley this form is getting huge now. Does everything here still feel like it should be under the *Edit requirements and location* title?

**Comment by ztolley** [comment]
Yes the form is big but yes it should be in this section

File: src/apps/investment-projects/middleware/forms/requirements.js (hunk 3 above)
Comment (ztolley) [reply src/apps/investment-projects/middleware/forms/requirements.js]: It can either be an array or a single string, so array would be incorrect

**Comment by MillsWest** [comment]
Hello. My suggestion was to keep it as plural then you don't need to do the (s) thing. 

Strategic drivers behind this investment 
then help text underneath in grey is
You can add more than one

**Comment by ztolley** [comment]
We don't do that anywhere else though, so it may be inconsistent.

**Comment by ztolley** [comment]
And currently, the styling doesn't support hints for 'add another', so it looks bad.

capture

So for consistency can we not use a hint and keep the wording used? It's obvious you can can have more than one because you have the 'add another' button

**Pull request merged**